tests/FSharp.Compiler.ComponentTests/ErrorMessages/ExpressionIgnoredRange.fs

```diff
@@ -0,0 +1,15 @@
+for i in 1 .. 10 do
+    printfn ""
+    printfn ""
+
+for i in 1 .. 10 do
+    printfn ""
+    printfn "" |> ignore
+    true
+
+for i in 1 .. 10 do
+    printfn ""
+    printfn "" |> ignore
+    {| A = 4 |}
+
+// TODO Add more cases
```

Comment:
How many warnings will we get now for

```fsharp
for i in 1 .. 10 do
    printfn ""
    3
    true
    {| A = 4 |}
```

?

(We currently get one for the whole sequential expression.)

Reply:
3 warnings. One for each non unit expression